ansible/roles/devspace/tasks/devspace-install.yml

```diff
@@ -100,79 +100,91 @@
 
 - name: set basic auth password for jenkins
   become: yes
   lineinfile:
-    dest: "{{ devhome }}/jenkins/conf.d/jenkins.conf"
+    dest: "{{ devspace_home }}/jenkins/conf.d/jenkins.conf"
     backup: yes
     regexp: "^        auth_basic_user_file /etc/nginx/conf.d/passwdfile;"
     line: "        auth_basic_user_file /etc/nginx/conf.d/passwdfile;"
     insertafter: "        auth_basic \"Restricted\";"
     state: present
-  when: jenkins_nginx_username|length>0 and jenkins_nginx_password|length>0
+  when: devspace_jenkins_nginx_username|length>0 and devspace_jenkins_nginx_password|length>0
 
 - name: set jenkins password
   become: yes
-  become_user: "{{ devuser }}"
+  become_user: "{{ devspace_user }}"
   htpasswd:
-    path: "{{ devhome }}/jenkins/conf.d/passwdfile"
-    name: "{{ jenkins_nginx_username }}"
-    password: "{{ jenkins_nginx_password }}"
-  when: jenkins_nginx_username|length>0 and jenkins_nginx_password|length>0
+    path: "{{ devspace_home }}/jenkins/conf.d/passwdfile"
+    name: "{{ devspace_jenkins_nginx_username }}"
+    password: "{{ devspace_jenkins_nginx_password }}"
+  when: devspace_jenkins_nginx_username|length>0 and devspace_jenkins_nginx_password|length>0
 
 - name: ssl cert for jenkins
   become: yes
-  become_user: "{{ devuser }}"
-  command: "{{ devhome }}/sslcert {{ devhome }}/jenkins/sslcert {{ openstack_ip }}"
+  become_user: "{{ devspace_user }}"
+  command: "{{ devspace_home }}/sslcert {{ devspace_home }}/jenkins/sslcert {{ devspace_openstack_ip }}"
   args:
-    chdir: "{{ devhome }}"
-    creates: "{{ devhome }}/jenkins/sslcert"
+    chdir: "{{ devspace_home }}"
+    creates: "{{ devspace_home }}/jenkins/sslcert"
 
 - name: ssl cert for nginx
   become: yes
-  become_user: "{{ devuser }}"
-  command: "{{ devhome }}/sslcert {{ devhome }}/nginx/sslcert {{ openstack_ip }}"
+  become_user: "{{ devspace_user }}"
+  command: "{{ devspace_home }}/sslcert {{ devspace_home }}/nginx/sslcert {{ devspace_openstack_ip }}"
   args:
-    chdir: "{{ devhome }}"
-    creates: "{{ devhome }}/nginx/sslcert"
+    chdir: "{{ devspace_home }}"
+    creates: "{{ devspace_home }}/nginx/sslcert"
 
-- name: rename to {{ omero_branch }}...
+- name: rename to {{ devspace_omero_branch }}...
   become: yes
-  become_user: "{{ devuser }}"
-  command: "{{ devhome }}/rename.py {{ omero_branch }}"
+  become_user: "{{ devspace_user }}"
+  command: "{{ devspace_home }}/rename.py {{ devspace_omero_branch }}"
   args:
-    chdir: "{{ devhome }}"
+    chdir: "{{ devspace_home }}"
+
+- name: fix jenkinsUrl
+  become: yes
+  become_user: "{{ devspace_user }}"
+  replace:
+    dest: "{{ devspace_home }}/home/{{ item }}"
+    regexp: '^(.*)<jenkinsUrl>(.*)</jenkinsUrl>'
+    replace: '\1<jenkinsUrl>https://{{ devspace_openstack_ip }}:8443</jenkinsUrl>'
+  with_items:
+    - jenkins.model.JenkinsLocationConfiguration.xml
+  when: devspace_openstack_ip|length>0
```

Comment:
I am sure `default(inventory_hostname)` would solve the problem but unfortunately I dont know how to use it